neuralmonkey/config/builder.py

```diff
@@ -26,21 +26,30 @@ def create(self):
 
         class_name = class_parts[-1]
 
-        # TODO should we not assume that everything is from neuralmonkey?
-        module_name = ".".join(["neuralmonkey"] + class_parts[:-1])
-
+        simple_module_path = ".".join(class_parts[:-1])
         try:
-            module = importlib.import_module(module_name)
-        except ImportError as exc:
-            # if the problem is really importing the module
-            if exc.name == module_name:
-                raise Exception(
-                    ("Interpretation '{}' as type name, module '{}' "
-                     "does not exist. Did you mean file './{}'? \n{}")
-                    .format(self.clazz,
-                            module_name, self.clazz, exc)) from None
-            else:
-                raise
+            import tensorflow as tf
+            module = importlib.import_module(simple_module_path)
+        except ImportError:
+            try:
+                if class_parts[0] == 'tf':
+                    # Due to the architecture of TensorFlow, it must be
+                    # imported this way.
+                    tf = importlib.import_module("tensorflow")
+                    module = getattr(tf, ".".join(class_parts[1:-1]))
+                else:
+                    module_name = ".".join(["neuralmonkey"] + class_parts[:-1])
+                    module = importlib.import_module(module_name)
+            except ImportError as exc:
```

Comment:
tohle mi přijde hodně divný, chytat `ImportError` v `except` bloku, kterej se raisne při import erroru. A předtim nefungovalo v konfiguraci tf? Podle mě by mělo fungovat, když se dá `import tensorflow as tf` a potom se volá `import_module` na modul z konfigurace.

A když ne, tak bych udělal pár regexpovejch pravidel, který budou preprocessovat vstup z konfiguráku (tedy `^tf\.` to nahradí za `tensorflow` a je to).